rpcs3/Emu/System.h

```diff
@@ -23,8 +23,8 @@ enum class video_renderer;
 
 enum class system_state : u32
 {
-	running,
 	stopped,
+	running,
```

Comment:
It would probably be wise to explicitly initialize m_system_state as stopped, instead of relying on its default value

Reply:
It is though, this is about a stage before the constructor takes place.

Reply:
Obviously. Doesn't change what I said though

Reply:
No I mean, before the constroctor takes place you can't initialize it. because it happens *before* the initialization itself. IIRC most compilers zero initialize it at compile time, and the ones which don't assign system_state::stopped at that stage. See https://en.cppreference.com/w/cpp/language/zero_initialization for more information.

Reply:
It still doesn't change what I said 🤣

Reply:
But we already initialize it explicitly, that's what I meant earlier.

Reply:
This change is already good, it makes sense to have empty/compiler-default state mean nothing is running. I suppose explicit init wouldn't hurt for existing code, but this is not a big deal breaker by itself.

Reply:
Yeah. Not a blocker. Just a precaution imo